server/routerlicious/packages/services-utils/src/abortSignal.ts

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { getGlobalAbortControllerContext } from "@fluidframework/server-services-client";
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+import type { Request, Response, RequestHandler, NextFunction } from "express";
+
+export function addAbortControllerForRequestMiddleware(): RequestHandler {
```

Comment:
Maybe just name this as `AbortContollerMiddelware` as this is a class like function?